### PR TITLE
Introduces proof documents.

### PR DESCRIPTION
**Pull request by oyeb**
Implementation plan:
- [x] basic schema
- [x] create endpoint `/api/proofs` (post)
- [x] use the [cloud storage SDK][sdk] to upload documents
- [x] add form elements to ppe create.
- [x] associate with availability
- [x] include proof upload in availability form

### CODE REVIEW

File: routes/proof.js [hunk 1]
```diff
@@ -29,7 +29,8 @@ function makeURI(proofKind, uri, file) {
  */
 const upload = multer({
   storage: multerGoogleStorage.storageEngine({
-    filename: (_req, file, cb) => { cb(null, `proofs/${nanoid(14)}-${file.originalname}`); }
+    acl:{},
```
Comment (oyeb) [inline routes/proof.js]: I'm very badass

File: migrations/20200402190917-create-proof.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        type: Sequelize.INTEGER
+      },
+      name: {
+        type: Sequelize.STRING,
+        allowNull: false
+      },
+      kind: {
+        allowNull: false,
+        type: Sequelize.ENUM,
+        values: ['document', 'hyperlink', 'manual'],
```
Comment (oyeb) [inline migrations/20200402190917-create-proof.js:18]: I'm not sure if we decided to keep `manual`  or drop it.

File: routes/index.js [hunk 3]
```diff
@@ -117,52 +120,52 @@ function findMatches(newPost, newPostType, mode) {
...
-      longitude: req.body.longitude,
-    }).then(function (created) {
-      // findMatches(created, 'Manufacturing', 'onCreate');
-      res.redirect('/ppe/map');
-    });
-  }
-
-
-})
+router.post('/ppe',
```
Comment (oyeb) [inline routes/index.js:123]: :heart_eyes:

**Review by oyeb — COMMENTED** [review]
Awesome! LGTM :bell:

File: migrations/20200402190917-create-proof.js (hunk 2 above)
Comment (shinjanxp) [reply migrations/20200402190917-create-proof.js:18]: I modified the schema to have Documents and Proofs. The Document model only has 'file' and 'hyperlink' as kind values. Manual has been removed.

**Pull request merged**